tests/unit/components/labelPrinting/LabelPrintingForm.spec.js

```diff
@@ -24,12 +19,14 @@ const evt = {
   },
 }
 
-const printerRequestFactory = RequestFactory('printers', false)
+const printersFactory = PrinterFactory()
 
 const plugins = [
   ({ store }) => {
     if (store.$id === 'root') {
-      store.api.v2.traction.printers.get = vi.fn().mockResolvedValue(printerRequestFactory.response)
+      store.api.v2.traction.printers.get = vi
+        .fn()
+        .mockResolvedValue(printersFactory.responses.fetch)
```

Comment:
I hope once we move everything to fetch, it will be just be printersFactory.responses.

Reply:
Maybe. I like the idea of it being descriptive and it gives us the flexibility of using different response types. I think we might need it if we do location tracking as Labwhere does not use JSON API resource.